editoast/src/models/fixtures.rs

```diff
@@ -113,8 +113,8 @@ pub fn simple_train_schedule_changeset(timetable_id: i64) -> Changeset<TrainSche
     Changeset::<TrainSchedule>::from(simple_train_schedule_base()).timetable_id(timetable_id)
 }
 
-pub fn simple_paced_train_changeset(timetable_id: i64) -> Changeset<PacedTrain> {
-    Changeset::<PacedTrain>::from(simple_paced_train_base()).timetable_id(timetable_id)
+pub fn simple_paced_train_changeset(timetable_id: i64) -> Changeset<PacedTrainModel> {
```

Comment:
```suggestion
pub fn simple_paced_train_changeset(timetable_id: i64) -> Changeset<models::PacedTrain> {
```

conveys the same information and avoids the indirection caused by the aliasing